cmd/snap-seccomp/main.go

```diff
@@ -771,6 +779,13 @@ func main() {
 		err = showSeccompLibraryVersion()
 	case "version-info":
 		err = showVersionInfo()
+	case "actlog-supported":
```

Comment:
Wonder if that should rather be part of version-info output. It conveys the information about capabilities of libseccomp-golang, and that in my view could count as version information the same way a libseccomp version and a hash of supported syscalls does.
Perhaps we could have a string like this:
```
<build-id> <libseccomp-version> <hash-of-syscall-names> <capability-tags>
```

Reply:
@bboozzoo - it would mean I could make one less callout to snap-seccomp in my subsequent PR, but I was a bit concerned I'd get bogged down in system-key. The 'daemon user' PR has spun so far out and I'm desperately trying to reign it in. Perhaps I can have the command return both, but have system-key only care about what it cares about now.